pkg/controllers/deprovisioning/controller.go

```diff
@@ -133,14 +134,14 @@ func (c *Controller) Reconcile(ctx context.Context, _ reconcile.Request) (reconc
 			continue
 		}
 		if cmd.action == actionRetry {
-			return reconcile.Result{Requeue: true}, nil
+			return reconcile.Result{RequeueAfter: immediately}, nil
 		}
 
 		// Attempt to deprovision
 		if err := c.executeCommand(ctx, d, cmd); err != nil {
 			return reconcile.Result{}, fmt.Errorf("deprovisioning candidates, %w", err)
```

Comment:
I'm not sure we should error here either. If an early deprovisioner experiences a transient error, it could cause back off and keep later ones from running.

Reply:
Great point

Reply:
We can always change the rate limiter for error backoff if we think that the error backoff mechanism is wrong right now